examples/compute_polar_coordinates.py

```diff
@@ -0,0 +1,362 @@
+"""Express 2D vectors in polar coordinates
+============================================
+
+Compute a vector representing head direction and express it in polar
+coordinates.
+"""
+
+# %%
+# Imports
+# -------
+
+# For interactive plots: install ipympl with `pip install ipympl` and uncomment
+# the following line in your notebook
+# %matplotlib widget
+import numpy as np
+from matplotlib import pyplot as plt
+
+from movement import sample_data
+from movement.io import load_poses
+from movement.utils.vector import cart2pol, pol2cart
+
+# %%
+# Load sample dataset
+# ------------------------
+# In this tutorial, we will use a sample dataset with a single individual
+# (a mouse) and six keypoints.
+
+ds_path = sample_data.fetch_dataset_paths(
+    "SLEAP_single-mouse_EPM.analysis.h5"
+)["poses"]
+ds = load_poses.from_sleap_file(ds_path, fps=None)  # force time_unit = frames
+
+print(ds)
+print("-----------------------------")
+print(f"Individuals: {ds.individuals.values}")
+print(f"Keypoints: {ds.keypoints.values}")
+
+
+# %%
+# The loaded dataset ``ds`` contains two data variables:``position`` and
+# ``confidence``. Both are stored as data arrays. In this tutorial, we will
+# use only ``position``:
+position = ds.position
+
+
+# %%
+# Compute head vector
+# ---------------------
+# To demonstrate how polar coordinates can be useful in behavioural analyses,
+# we will compute the head vector of the mouse.
+#
+# We define it as the vector from the midpoint between the ears to the snout.
+
+# compute the midpoint between the ears
+midpoint_ears = 0.5 * (
+    position.sel(keypoints="left_ear") + position.sel(keypoints="right_ear")
+)
+
+# compute the head vector
+head_vector = position.sel(keypoints="snout") - midpoint_ears
+
+# drop the keypoints dimension
+head_vector = head_vector.drop_vars("keypoints")
+
+# %%
+# Visualise the head trajectory
```

Comment:
I wonder if this is a good opportunity to showcase the ability to plot the frame under the trajectory (now that we have the frame for this dataset).
You could first load and plot the image with maptlotlib, then overplot the trajectory over it. It will also be a good visual check that the coordinates of the trajectory match the image, and the reason for the cruciform shape will be obvious.

You can get the frame path in a similar way to the poses path:
```python
frame_path = sample_data.fetch_dataset_paths("SLEAP_single-mouse_EPM.analysis.h5")["frame"]
```
If you had loaded the sample data the "normal" way, i.e. via `fetch_dataset`, the frame path would already be among the dataset attributes.